spec/mail/utilities_spec.rb

```diff
@@ -403,7 +404,7 @@
     it "should say an empty string is blank" do
       expect(Mail::Utilities.blank?('')).to be_truthy
       expect(Mail::Utilities.blank?(" ")).to be_truthy
-      a = ''; 1000.times { a << ' ' }
+      a = String.new(''); 1000.times { a << ' ' }
```

Comment:
ok -- but not sure this change really has anything to do with a PR thats about frozen string literals